geetools/ee_dictionary.py

```diff
@@ -80,3 +82,172 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(self, valueType: ee.List | ee.Dictionary | Any = Any) -> ee.FeatureCollection:
+        """Convert a :py:class:`ee.Dictionary` to a :py:class:`ee.FeatureCollection` with no geometries (table).
+
+        There are 3 different type of values handled by this method:
+
+        1. Any (default): when values are a `ee.String` or `ee.Number`, the
+        keys will be saved in the column "system:index" and the values in the
+        column "value".
```

Comment:
In rst, the content of a paragraph should meet the indetnation after the "x." or "-". In your case that would do something like: 

```suggestion
        1. Any (default): when values are a `ee.String` or `ee.Number`, the
           keys will be saved in the column "system:index" and the values in the
           column "value".
```